src/averageAges.js

```diff
@@ -14,12 +14,16 @@
  *
  * @return {number}
  */
-function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+function calculateMenAverageAge(people, century = 0) {
+  const menList = people.filter(person => century > 0
+    ? person.sex === 'm' && Math.ceil(person.died / 100) === century
+    : person.sex === 'm');
+  let MenAverageAge = menList.reduce((sumAgeMan, currenAgeMan) =>
+    sumAgeMan + (currenAgeMan.died - currenAgeMan.born), 0);
+
+  MenAverageAge = MenAverageAge / menList.length;
+
+  return MenAverageAge;
```

Comment:
Avoid var reassigning
```suggestion
  const menAverageAge = menList.reduce((sumAgeMan, currenAgeMan) =>
    sumAgeMan + (currenAgeMan.died - currenAgeMan.born), 0);

  return menAverageAge / menList.length;;
```